packages/patternfly-3/patternfly-react-extensions/package.json

```diff
@@ -65,7 +65,7 @@
   },
   "peerDependencies": {
     "prop-types": "^15.6.1",
-    "react": "^16.3.2",
-    "react-dom": "^16.3.2"
+    "react": "16.4.0",
+    "react-dom": "16.4.0"
```

Comment:
@redallen we don't want to change peer dependencies, correct?

Reply:
Correct, let's leave these alone.

Nice new icon btw @boaz0 !